app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```

Comment:
ruby file, use double-quotes

Reply:
i have changed ' to " yesterday but it turnoed out the same.  i will change again